doc/source/preparing_releases_and_hotfixes.rst

```diff
@@ -205,10 +210,11 @@ Once the release is completed you can go ahead and update the ``develop`` branch
 #. Once these changes are merged into the ``develop`` branch, message the developers on discord and GitHub Discussions letting them know that the feature freeze is over.
 
 
-Clean up old developer builds of the documentation
---------------------------------------------------
+.. _Clean up old development builds:
+Clean up old developement builds
+--------------------------------
 
-Whilst new docs are automatically deployed on a release, old developer builds (appended with ``-dev``) are not automatically cleaned up. To avoid causing large amounts of files being uploaded to GitHub Pages, we need to delete these old developer builds manually. To do this switch to the ``gh-pages`` branch, delete these old files, and push the change directly. You should do this for both the core library and the UserGuide.
+Whilst new docs are automatically deployed on a release, old developer builds (appended with ``-dev``) are not automatically cleaned up. To avoid causing large amounts of files being uploaded to GitHub Pages, we need to delete these old developer builds manually. To do this switch to the ``gh-pages`` branch, make sure it is up to date with upstream (i.e., run ``git pull`` ), delete the old files, and push the change directly. You should do this for both the core library and the UserGuide.
```

Comment:
```suggestion
Whilst new docs are automatically deployed on a release, old developer builds (appended with ``-dev``) are not automatically cleaned up. To avoid causing large amounts of files being uploaded to GitHub Pages, we need to delete these old developer builds manually. To do this switch to the ``gh-pages`` branch, make sure it is up to date with upstream (i.e., run ``git pull`` ), delete the old files, and push the change directly. You should do this for both the core library and the UserGuide.

In addition, update `versions.json` to remove the old `-dev` entry.
```